src/context/index.tsx

```diff
@@ -40,4 +40,4 @@ export const useAppContext = () => {
     }
     return context || {}
 }
-export default AppWrapperContext
```

Comment:
Nit: Why does this file show changes over here, as you have done no changes to this line.

Reply:
I am not sure why this happened, there is no changes in that line.